app/components/romanNumbers/romanNumbers.controller.spec.js

```diff
@@ -1,5 +1,68 @@
+
 'use strict';
-describe('RomanNumberController', function () {
 
-});
+describe('RomanNumbersController', function () {
+
+    beforeEach(module('myApp.components.romanNumbers'));
+
+    var RomanNumbersController, $scope, romanNumeralTranslatorService, $event;
+
+    beforeEach(inject(function ($rootScope, $controller, _romanNumeralTranslatorService_) {
+        $scope = $rootScope.$new();
+
+        RomanNumbersController = $controller('RomanNumbersController', {
+            $scope: $scope
+        });
+
+        romanNumeralTranslatorService = _romanNumeralTranslatorService_;
+
+    }));
+
+    describe('translate Function', function () {
+
+        it("should set romanInteger to empty string if decimalInteger is empty", function () {
+            RomanNumbersController.translate();
+
+            expect(RomanNumbersController.romanInteger).toBe('');
+        });
+
+        it("should check if decimalInteger is having some value and should assign equivalent roman value with romanInteger", function () {
```

Comment:
describe to me the behavior, not the implementation.  Keep the implementation a black box.